{{cookiecutter.repo_name}}/requirements/local.txt

```diff
@@ -4,5 +4,5 @@
 
 django-debug-toolbar
 docker-compose >= 1.7.1
-git+https://github.com/edx/i18n-tools.git@v0.3.1#egg=i18n_tools==0.3.1
+edx-i18n-tools >= 0.3.3
```

Comment:
@nedbat I was basing the style here on the way docker-compose was being used. Perhaps Clinton can comment on it more when he is back online